extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -54,3 +54,29 @@ export function convertNumToTwoDecimalStringInMB(value: number): string {
 export function escapeSingleQuotes(value: string): string {
 	return value.replace(/'/g, "\'");
 }
+
+/**
+ * Backup media device type: https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.management.smo.devicetype
+ */
+export enum MediaDeviceType {
```

Comment:
The constants.ts file in the same folder would probably be a better place for these.